### ISSUE TITLE
Intel: Incorrect thread count with Pentium Silver N5000

### ISSUE THREAD

**Issue by dd86k** [guitarxhero/ddcpuid#28]
I'm currently getting incorrect thread count on my Pentium Silver N5000:
```
Vendor      : GenuineIntel
Brand       : Intel(R) Pentium(R) Silver N5000 CPU @ 1.10GHz
Identifier  : Family 6 (0x6) [0x6:0x0] Model 122 (0x7a) [0xa:0x7] Stepping 1
Cores       : 32 threads
```

CPUID dump attached

[cpuid_dump.log](https://github.com/dd86k/ddcpuid/files/7132630/cpuid_dump.log)

**Comment by dd86k** [comment]
Fixed in 77fba93.

Had to support CPUID.0Bh.

**Comment by dd86k** [comment]
To be released in 0.19.
